### PR TITLE
Rename attributes and default styles.

### LINKED ISSUE #1475: Customized styling improvements
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Would you like to work on the issue?**
Please state if this issue should be assigned to you or who you think could help to solve this issue.

### PR DESCRIPTION
**Pull request by santosh-pingle**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes #1475 

**Description**

- Rename default styles and attributes in datacapture module.
- try to differentiate style and attributes names from android default names.

**Alternative(s) considered**
Have you considered any alternatives? And if so, why have you chosen the approach in this PR?

**Type**
Choose one:  Code health 

**Screenshots (if applicable)**

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md).
- [x] I have read the [Contributing](https://github.com/google/android-fhir/wiki/Contributing) page.
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate ).
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach.
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project.
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally.
- [x] I have built and run the demo app(s) to verify my change fixes the issue and/or does not break the demo app(s).

### CODE REVIEW

**Review by juanchazo — APPROVED** [review]
Nice, these are definitely more concise and clear! 💯

**Comment by santosh-pingle** [comment]
> Nice, these are definitely more concise and clear! 💯

@juanchazo thanks!
@jingtang10 
Can you please also review

File: datacapture/src/main/res/values/attrs.xml [hunk 1]
```diff
@@ -16,26 +16,21 @@
 -->
 <resources>
     <attr name="groupHeaderTextAppearanceQuestionnaire" format="reference" />
```
Comment (jingtang10) [inline datacapture/src/main/res/values/attrs.xml:18]: why is this not changed?

**Review by jingtang10 — CHANGES_REQUESTED** [review]

File: datacapture/src/main/res/values/attrs.xml (hunk 1 above)
Comment (santosh-pingle) [reply datacapture/src/main/res/values/attrs.xml]: removed it, as not referenced in the code.

[the remark this answers is not in the document]
Comment (santosh-pingle) [reply datacapture/src/main/res/values/attrs.xml]: Updated change.

File: datacapture/src/main/res/layout/questionnaire_item_date_picker_view.xml [hunk 2]
```diff
@@ -44,7 +44,7 @@
 
         <com.google.android.material.textfield.TextInputEditText
             android:id="@+id/text_input_edit_text"
-            style="?attr/editTextInputTextAppearanceQuestionnaire"
+            style="?attr/questionnaireInputTextEditTextStyle"
```
Comment (jingtang10) [inline datacapture/src/main/res/layout/questionnaire_item_date_picker_view.xml:47]:
```suggestion
            style="?attr/questionnaireTextInputEditTextStyle"
```

File: datacapture/src/main/res/values/attrs.xml [hunk 3]
```diff
@@ -15,27 +15,45 @@
...
-        name="dropDownSelectedTextAppearanceQuestionnaire"
-        format="reference"
-    />
-    <attr name="dropDownStyleQuestionnaire" format="reference" />
-    <attr name="editTextInputTextAppearanceQuestionnaire" format="reference" />
-    <attr name="editTextLayoutStyleQuestionnaire" format="reference" />
-    <attr name="chipStyleQuestionnaire" format="reference" />
-    <attr name="buttonStyleQuestionnaire" format="reference" />
-    <attr name="inputErrorTextAppearanceQuestionnaire" format="reference" />
+    <!--   Style for question text. -->
```
Comment (jingtang10) [inline datacapture/src/main/res/values/attrs.xml:18]: Maybe separate these styles into 2 different sections.
Add a header comment for styles for widgets

and then add another header comment for styles for navigation (submit, prev next buttons)

**Review by jingtang10 — APPROVED** [review]

**Review by jingtang10 — APPROVED** [review]

**Pull request merged**